apps/web-giddh/src/app/multi-currency-reports/directives/financial-accordion.directive.ts

```
@@ -0,0 +1,43 @@
+
+@Directive({
+    selector: '[financial-accordion]'
+})
+export class FinancialAccordionDirective {
+    // tslint:disable-next-line:no-input-rename
+    @Input('financial-accordion') public data: ChildGroup;
+
+    // tslint:disable-next-line:no-input-rename
+    constructor() {
+        
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)